lively.ide/world.js

```diff
@@ -1140,6 +1140,11 @@ export class LivelyWorld extends World {
     }]);
 
     items.push(['Remove Morph', () => self.abandon(true)]);
+    items.push(['Rename Morph', async () => {
```

Comment:
Literally quoting from the commit message that introduced this ... :

> **remove renaming option from double-clicking on node in scene-graph**
Instead, renaming is now available via the context menu of a morph and thus in the scene graph as well, as the menu of a scene graph node map to the menu of the morph the node represents.

Basically, I moved the (non-working) renaming flow for a morph that was implemented by double-clicking in the scene graph into the morph menu, which has the same accessibility through the scene graph, as the menu that opens on the nodes is actually the menu of the morph represented by the node and hat the actual morph as its target.

---
As I mentioned on Friday, I will create a new PR (including these changes) when I am done so that you can actually review.
However, not all fixes are there yet.

Reply:
Gotcha.